### ISSUE TITLE
P3 unblocked: use python -m cdlib.cli.benchmark

### ISSUE THREAD

**Issue by KusalPabasara** [Deep-NeuralNetworks-Research-Project/metrics-integration-kusal#7]
\`cdlib.cli.benchmark\` is in this repo and supersedes \`scripts/benchmark_encoders.py\`.

\`\`\`bash
python -m cdlib.cli.benchmark --model dummy --img-size 256
\`\`\`

Disclosure fields: GPU name every run, AMP/TF32, batch size, median+IQR (CUDA events, warm-up excluded), pair-input FLOPs via \`torch.utils.flop_counter\` (true FLOPs; MACs=FLOPs/2), peak reserved vs allocated memory.

Real encoder keys need P3 modules or the team monorepo. Until then, \`--model dummy\` is the smoke path.

**Comment by KusalPabasara** [comment]
**Done on the P5 side.**

`python -m cdlib.cli.benchmark --model dummy --img-size 256` is the smoke path. Disclosure fields (GPU name, AMP/TF32, batch, median+IQR, pair FLOPs, peak reserved vs allocated) are implemented.

@weenukarajapaksha / @Januda-lelwala — real encoder keys still need your modules or the monorepo. Until then `--model dummy` is enough to unblock encoder timing experiments against the CLI contract.

Closing this issue.
